AuthenticatorShared/UI/Platform/Settings/Settings/SettingsView.swift

```diff
@@ -41,8 +41,19 @@ struct SettingsView: View {
         switch store.state.biometricUnlockStatus {
         case let .available(type, enabled: enabled, _):
             SectionView(Localizations.security) {
-                VStack(spacing: 0) {
+                VStack(spacing: 8) {
                     biometricUnlockToggle(enabled: enabled, type: type)
+                    SettingsMenuField(
```

Comment:
🎨 This should set `hasDivider: false` so the divider doesn't show up on the row

Reply:
🤦‍♂️ Good catch!